machine/v1beta1/types_awsprovider.go

```diff
@@ -91,6 +91,10 @@ type AWSMachineProviderConfig struct {
 	// +kubebuilder:validation:Maximum:=7
 	// +optional
 	PlacementGroupPartition *int32 `json:"placementGroupPartition,omitempty"`
+	// capacityReservationId specifies the target Capacity Reservation into which the instance should be launched.
+	// This must be a string value.
```

Comment:
Any more definition than this? Characters, length?

Reply:
AWS IDs always start with short name of resource_name ex: cr-**. I updated the comment to
```The field size should be greater than 0 and the field input must start with cr-***
```
I hope this is fine.